source/math/FloatBox2D.ooc

```diff
@@ -60,6 +60,9 @@ FloatBox2D: cover {
 	pad: func ~fraction (pad: Float) -> This {
 		this pad(pad * this size / 2.0f)
 	}
+	pad: func ~fractionAverage (pad: Float) -> This {
```

Comment:
I don't like the name. Suffixes are meant to help the C compiler disambiguate between function with the same name, but with a different signature. Now you have 2 `pad` functions with the same signature, but with different suffixes. Come up with a real name for this function instead.